cps/tasks/download.py

```diff
@@ -80,9 +80,8 @@ def run(self, worker_thread):
                                     last_progress_time = datetime.now()
                     else:
                         elapsed_time = (datetime.now() - last_progress_time).total_seconds()
-                        if elapsed_time >= fragment_stuck_timeout:
-                            self.message += f"<br>Some fragments are taking longer than expected to download. Please wait..."
-
+                        if elapsed_time >= timeout:
+                            self.message = f"{self.media_url_link} is taking longer than expected. It could be a stuck download due to unavailable fragments (<a href='https://github.com/yt-dlp/yt-dlp/issues/2137' target='_blank'>yt-dlp/yt-dlp#2137</a>) and/or an error in xklb's media check. Please wait as we try again. See <a href='https://github.com/iiab/calibre-web/pull/223' target='_blank'>#223</a> for more info."
```

Comment:
@deldesir @avni is the revision below more accurate?  (Or if I'm wrong, LMK!)

```suggestion
                            self.message = f"{self.media_url_link} is taking longer than expected. It could be a stuck download due to unavailable fragments (<a href='https://github.com/yt-dlp/yt-dlp/issues/2137' target='_blank'>yt-dlp/yt-dlp#2137</a>) and/or an error in xklb's media_check. Please wait as we keep trying. See <a href='https://github.com/iiab/calibre-web/pull/223' target='_blank'>#223</a> for more info."
```

Reply:
"keep trying" is more accurate, yes! TY!

Reply:
@deldesir please revise at any time (e.g. with a new PR) if improvement's helpful 💯

Reply:
Merging of this PR is now cleaned up with this separate commit:

- https://github.com/iiab/calibre-web/commit/23af2b0db4377ba9a048f35036202a43840b1537

(Apologies I messed up, and didn't get that revision from ~30min ago properly into this PR!)